installer/install.iss

```diff
@@ -54,7 +54,7 @@ SolidCompression=yes
 #endif
 SourceDir={#SOURCE_DIR}
 #if BITNESS=='64' || INSTALLER_FILENAME_SUFFIX=='arm64'
-ArchitecturesInstallIn64BitMode=x64 arm64
+ArchitecturesInstallIn64BitMode=x64compatible arm64
```

Comment:
With the new `x64compatible`, there should be [no need](https://jrsoftware.org/ishelp/index.php?topic=setup_architecturesinstallin64bitmode) to explicitly set `arm64` anymore:

> **x64compatible**
> Matches systems capable of running x64 binaries. This includes systems running x64 Windows, and also Arm64-based Windows 11 systems, which have the ability to run x64 binaries via emulation.

Reply:
Makes sense to me!